pyproject.toml

```
@@ -0,0 +1,49 @@
+[build-system]
+requires = ["setuptools>=75", "setuptools_scm>=7", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "python-stripzip"
+description = "strip zip timestamps for reproducible builds"
+readme = "README.rst"
+requires-python = ">=3.8"
+license = "MIT"
+license-files = ["LICENCE.md"]
```

the current form doesn't work in the python3.8 environment so this may work around it
```suggestion
license = {text = "MIT"}
```